portafly/portafly_api.yml

```
@@ -0,0 +1,127 @@
+---
+openapi: 3.0.2
+info:
+  title: PortaFly API
+  version: 1.0.0
+  description: A tuned API for PortaFly, a Patternfly SPA consuming 3scale Porta API
+    MGMT solution
+  license:
+    name: MIT License
+    url: https://opensource.org/licenses/MIT
+paths:
+  /applications:
+    summary: Returns the list of applications across all services.
+    description: |+
+      Returns the list of applications across all services. Note that applications are scoped by service and can be paginated.
+
+    get:
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Application'
+              examples:
+                Application List Example:
+                  value:
```

We could benefit from referencing our examples:
```suggestion
                  $ref: '#/components/examples/applicatoinList'
```
Then later:
```yaml
...
components:
  examples:
    applicatoinList:
      summary: 'a list of applications'
      value:
        - id: "1"
        ...
        - id: "2"
        ...
```